habitat_baselines/evaluate_ppo.py

```diff
@@ -207,7 +194,15 @@ def eval_checkpoint(checkpoint_path, args, writer, cur_ckpt_idx=0):
 
             # episode ended
             if not_done_masks[i].item() == 0:
-                stats_episodes.add(current_episodes[i].episode_id)
+                stats_episodes[current_episodes[i].episode_id] = infos[i][
```

Comment:
Let's just make this a dictionary of dictionaries.  Then you can keep both SPL and reward in the dictionary and you can use the length of the dictionary to keep track of the number of finished episodes.